user-org-service/app/controllers/healthmanager/HealthController.java

```diff
@@ -24,6 +26,10 @@
      */
 
     public CompletionStage<Result> getHealth() {
+
+        ProjectLogger.log(
+                "HealthController :getHealth() : in HealthController ", LoggerEnum.INFO.name());
```

Comment:
it should have method name only no brackets.
For method entry log should be :
("className:methodname:call started at timeinmilles",LoggerEnum.debug.name());
for method end:
("className:methodname:call ended at timeinmilles",LoggerEnum.debug.name());